src/process.c

```diff
@@ -33,6 +35,17 @@ void process_init() {
 
     current->state = PROCESS_STATE_READY;
 
+    // establish initial permissions
+    struct process_permissions *initial_permissions = kmalloc(sizeof(struct process_permissions));
+    initial_permissions->max_number_of_pages = memory_pages_total();
+    // initial_permissions.directories =
+    initial_permissions->max_width = 640; // temporary
```

Comment:
They weren't public and I wanted a second opinion on whether to make them such; now I shall